gasstation/gasstattion.go

```diff
@@ -51,14 +63,13 @@ func (gs *GasStation) SuggestGasPrice() (uint64, error) {
 		if len(blk.Actions) == 0 {
 			continue
 		}
-
+		if gs.IsSystemAction(blk.Actions[0]) && len(blk.Actions) == 1 {
```

Comment:
```suggestion
		if len(blk.Actions) == 1 && gs.IsSystemAction(blk.Actions[0]) {
```

Short circuit: https://kuree.gitbooks.io/the-go-programming-language-report/content/22/text.html